auth0_flutter/windows/test/id_token_signature_validator_test.cpp

```
@@ -672,46 +670,60 @@ TEST(IdTokenSignatureValidatorTest, Step3FailureThrowsIdTokenValidationException
-// Destructor closes the listener; any close() error is swallowed so that test
+// Destructor stops the server and joins its background thread so that test
 // teardown is always clean.
 // ---------------------------------------------------------------------------
 
 class TestJwksServer
 {
 public:
-    TestJwksServer(int port, web::http::status_code status, const std::string &body)
-        : uri_("http://127.0.0.1:" + std::to_string(port) + "/jwks.json"),
-          listener_(utility::conversions::to_string_t(uri_))
+    TestJwksServer(int port, int status, const std::string &body)
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: server_

(cppcoreguidelines-pro-type-member-init)